lib/components/sidebar/sidebar.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Drawer as AntDrawer } from 'antd';
+import Container from './container';
+import styles from './sidebar.css';
+
+const Sidebar = React.memo(
+  ({ visibleDrawer, children, width, placement, onClose, closable, style }) => {
```

Comment:
I don't think we should need to pass in `width` or `style`. We want all of them to look the same wherever this component is used.

Nitpick: Also, can we rename `visibleDrawer` to `visible`?

Reply:
fixed